spiped/README

```diff
@@ -21,9 +21,12 @@ Options:
 		[ipv6::addr]:port
 	Note that hostnames are resolved when spiped is launched and are not
 	re-resolved later; thus if DNS entries change spiped will continue to
-	connect to the expired address.
+	accept at the expired address.
     -t <target socket>
-	Address to which spiped should connect.
+	Address to which spiped should connect. The accepted formats are the
+	same as the ones accepted by <source socket>.
```

Comment:
My apologies, I just noticed that this is a README, not a man-page.  I should have made the comments about the man-page, although by symmetry the changes should probably also be made in the README, so the basic idea remains the same.